ovh/provider.go

```diff
@@ -229,7 +227,7 @@ func Provider() *schema.Provider {
 			"ovh_dedicated_nasha_partition":                                  resourceDedicatedNASHAPartition(),
 			"ovh_dedicated_nasha_partition_access":                           resourceDedicatedNASHAPartitionAccess(),
 			"ovh_dedicated_nasha_partition_snapshot":                         resourceDedicatedNASHAPartitionSnapshot(),
-			"ovh_dedicated_server_install_task":                              resourceDedicatedServerInstallTask(),
+			"ovh_dedicated_server_reinstall_task":                            resourceDedicatedServerInstallTask(),
```

Comment:
we should avoid this change as it unnecessarily breaks current usages

Reply:
I don't understand why a pull request was created...
I'm not ready yet :)
You will see that this breaking change is necessary to get ride of the personal template dependencies (/me/installationTemplate)...